.github/fixtures/test-release-statistics/cliff.toml

```
@@ -0,0 +1,48 @@
+# git-cliff ~ configuration file
+# https://git-cliff.org/docs/configuration
+
+[changelog]
+# A Tera template to be rendered for each release in the changelog.
+# See https://keats.github.io/tera/docs/#introduction
+body = """
+{% if version %}\
+    ## [{{ version | trim_start_matches(pat="v") }}] - {{ timestamp | date(format="%Y-%m-%d") }}
+{% else %}\
+    ## [unreleased]
+{% endif %}\
+{% for group, commits in commits | group_by(attribute="group") %}
+    ### {{ group | upper_first }}
+    {% for commit in commits %}
+        - {{ commit.message | upper_first }}\
+    {% endfor %}
+{% endfor %}
+### Commit Statistics
+    * {{ statistics.commit_count }} commits contributed to the release.
+	{%- if statistics.commit_duration_days is defined %}
+	* {{ statistics.commit_duration_days }} days spanned between the first and last commit.
```

This sounds better imo:

```suggestion
	* {{ statistics.commit_duration_days }} days passed between the first and last commit.
```